lib/jsdap.js

```diff
@@ -112,7 +112,8 @@ export default class JsDap {
                         callback(dataset);
                     },
                     false,
-                    extraheaders
+                    extraheaders,
```

Comment:
I am thinking maybe better to have an option object, which includes extreaHeaders and withCredentials,  also the timeout, but the code dosen't handle timeout yet